source/community/index.html.haml

```diff
@@ -31,14 +31,14 @@ title: Community
       * [Report an issue](issues)
       * [Propose a feature in the forum](http://talk.manageiq.org/c/developers/blueprints)
       * [View the ManageIQ Roadmap on Trello](https://trello.com/b/UeTqKlp3/manageiq-roadmap)
-      
+
   %section.col-md-4
     :markdown
       ### Develop
 
       * [Developer Documentation](/documentation/development/)
       * [Upgrade an image in-place](update-image)
-      * [Install and deploy, from source](install-from-source)
+      * [Install and deploy, from source](https://github.com/ManageIQ/guides/blob/master/developer_setup.md)
```

Comment:
guides is already mirrored to the website, so probably better to point to the website link @ /documentation/development/developer_setup